style.css

```diff
@@ -1,3 +1,220 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
-  background-color: blueviolet;
+  background-color: #eee;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+}
+
+.container {
+  width: 80%;
+  margin: 0 auto;
+}
+
+/* ---global css--- */
+
+.white-color {
+  color: #fff;
+}
+
+.parent-element {
+  position: relative;
+}
+
+.child-element {
+  position: absolute;
+}
+
+.image-width {
+  width: 364px;
+  height: 200px;
+}
+
+.no-text-decoration {
+  text-decoration: none;
+}
+
+.footer-text-size {
+  font-size: 0.66em;
+}
+
+/* ---header--- */
+
+header {
+  height: 546px;
+  background: url("images/background.jpg") no-repeat 50% 100%;
+}
+
+/* ---top navigation--- */
+
+nav {
+  background: linear-gradient(rgba(17, 17, 17, 0.307) 50%, rgba(141, 137, 137, 0.803) 50%);
```

Comment:
- [ ] It should be darker like the original (but keep the linear-gradient since it is in the requirements).


| Yours | Original |
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/61479648/92411818-c6371400-f11f-11ea-86ef-fbfc9456eee7.png) |![image](https://user-images.githubusercontent.com/61479648/92411825-cdf6b880-f11f-11ea-8880-2001c784bed9.png) |